blog/2023-11-01-feature-flags-sdks-architectures.md

```diff
@@ -0,0 +1,159 @@
+---
+slug: 'feature-flags-sdks-architectures'
+title: 'Feature Flags services Server-side SDKs different architectures approaches'
+date: 2023-11-01
+authors: [liran2000]
+description: "Feature Flags services Server-side SDKs different architectures approaches"
+tags: [client, sdk]
+draft: false
+---
+
+Server-side feature flag software development kits (SDKs) are common for letting applications like microservices interact with Feature flag services.
+Feature Flags SDKs have several used functionalities, mainly feature flag evaluation with or without context.
+Each Feature Flags service can publish SDKs in multiple programming languages.
+The Feature Flags services commonly expose APIs via endpoints like REST HTTP and/or gRPC.
+SDKs are an important layer, as network traffic loads affect both the application and the Feature Flags services.
+When the Feature Flags service is a Cloud SaaS service, some services support a relay proxy or a sidecar application, which can be deployed on an organization network.
+The Relay Proxy lets multiple clients connect to a local proxy, reducing the number of outbound connections to the Cloud Service.
+Considering that a large amount of microservices using SDK instances can be deployed, this can be significant.
+
+
+<!--truncate-->
+
+## Feature Flags services Server-side SDKs different architectures approaches
+
+During usage and development involvement at several feature flag services providers, I have encountered some different
+architectures approaches used by some vendors. Each vendor server-side SDK can commonly belong to one of the described
+approaches.
```

Comment:
can we change lines 25 - 27 to the following:
Through my own personal use as well as helping to develop several feature flag service providers, I have found that there are a few architectural approaches that most vendors server-side SDKs typically fall under.